eng/Packages.Data.props

```diff
@@ -153,7 +155,7 @@
     All should have PrivateAssets="All" set so they don't become package dependencies
   -->
   <ItemGroup>
-    <PackageReference Update="Microsoft.Azure.AutoRest.CSharp" Version="3.0.0-beta.20220623.2" PrivateAssets="All" />
+    <PackageReference Update="Microsoft.Azure.AutoRest.CSharp" Version="3.0.0-beta.20220602.1" PrivateAssets="All" />
```

Comment:
This impacts the entire repository and all libraries.   This should not be changed.